common/interfaces/user_interface/ui_multi_instruction.c

```diff
@@ -364,7 +387,7 @@ void multi_instruction_init(const char **arr, const uint8_t count, const uint16_
 
     for (uint8_t i = 0; i < count; i++)
     {
-        snprintf(data->strings[i], sizeof(data->strings[i]), "%s", arr[i]);
+        snprintf(data->instruction_content[i].text, sizeof(data->instruction_content[i].text), "%s", arr[i]);
```

Comment:
set other fields, for example `instruction_content[x].img = NULL`

Reply:
Will add a memzero for the `data` object

Reply:
added